docs/security/trust/content_trust.md

```diff
@@ -0,0 +1,262 @@
+ and tag combination many times changing the image with each build. 
+
+Trust applies to the `TAG` portion of an image. Each image repository has a set
+of keys that image creators use to sign an image tag. Image creators have
+discretion on which tags they sign.
+
+An image repository can contain an image with one tag that is signed and another
+tag that is not. For example, consider [the Mongo image
+repository](https://registry.hub.docker.com/_/mongo/tags/manage/). The `latest`
+tag could be unsigned while the `3.1.6` tag could be signed. It is the
+responsibility of the image creator to determine if an image tag is signed or
+not. In this representation, some image tags are signed, others are not:
+
+![Signed tags](../images/tag_signing.png)
+
+Currently, it is also possible to sign some image tag pushes and not others. For
+example, a creator can push a tagged image `someimage:latest` and sign it.
+Later, a creator can push and not sign `someimage:latest`. This is useful if
+creators want to iterate over a image before signing it's tag officially.
+
+Image consumers can enable trust to ensure that images they use were signed. If
+a consumer enables trust, they can only pull or run trusted images. Enabling
+trust is like wearing a pair of rose-colored glasses. Consumers "see" only
+signed images tags and the less desirable, unsigned image tags are "invisible" to them.
+
+![Trust view](../images/trust_view.png)
+
+To the consumer who is not using trust, every image is visible and can be consumed.
+
+
+### Trust operations and keys
+
+When trust is enabled, `docker` CLI commands that operate on tagged images must
+either have content signatures or explicit content hashes. The commands that
+operate with trust are:
+
+* `push`
+* `build`
+* `create` 
+* `pull`
+* `run` 
+
+Trust for an image tag is managed through the use of keys. Docker's content
+trust makes use four different keys: 
+
+| Key                 | Description                                                                                                                                                                                                                                                                                                                                                                         |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| offline key            | Root of trust for a image tag. When trust is enabled, you create this key once. Then, you can use this key to create tagging keys. |
+| target and snapshot | These two keys are known together as the "repository" key. When trust is enabled, you create this key when you add a new image repository. If you have the offline key, you can export the tagging key and allow other images creator that sign image tags.    |
```

Comment:
You don't need the offline key to export the tagging key. As long as you have the tagging key you can always export it and allow other trusted publishers to sign tags for your repository.